src/pages/AssignServerRoles.js

```diff
@@ -1373,50 +1424,67 @@ class AssignServerRoles extends BaseWizardPage {
+            <ActionButton displayLabel={translate('add.server.set.network')} type='default'
+              clickAction={() => this.setState({showBaremetalSettings: true})}/>
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  //gloria
   render() {
     let serverId = (this.state.activeRowData && this.state.activeRowData.id) ? this.state.activeRowData.id : '';
+    const contentClass =
+      'wizard-content' + (this.props.mode && this.props.mode === 'addserver' ? ' smaller-margin' : '');
     return (
       <div className='wizard-page'>
-        <EditCloudSettings
-          show={this.state.showCloudSettings}
-          onHide={() => this.setState({showCloudSettings: false})}
-          model={this.props.model}
-          updateGlobalState={this.props.updateGlobalState}/>
-        <div className='content-header'>
-          <div className='titleBox'>
-            {this.renderHeading(translate('add.server.heading'))}
-          </div>
-          <YesNoModal show={this.state.showDeleteServerConfirmModal} title={translate('warning')}
-            yesAction={this.deleteServer}
-            noAction={() => this.setState({showDeleteServerConfirmModal: false})}>
-            {translate('server.delete.server.confirm', serverId)}
-          </YesNoModal>
-          <YesNoModal show={this.state.showImportServerConfirmModal} title={translate('warning')}
-            yesAction={this.saveImportedServers}
-            noAction={() => this.setState({showImportServerConfirmModal: false, importedResults: {}})}>
-            {translate('server.import.server.confirm')}
-          </YesNoModal>
-          <div className='buttonBox'>
-            <div className='btn-row'>
-              <ActionButton displayLabel={translate('edit.cloud.settings')} type='default'
-                clickAction={() => this.setState({showCloudSettings: true})} />
-              <ActionButton displayLabel={translate('add.server.set.network')} type='default'
-                clickAction={() => this.setState({showBaremetalSettings: true})}/>
-            </div>
-          </div>
-        </div>
-        <div id='AssignServerRoleId' className='wizard-content'>
+        {this.props.mode === undefined && this.renderCloudSettings()}
+        {this.props.mode === undefined && this.renderInstallContentHeading()}
+        <YesNoModal show={this.state.showDeleteServerConfirmModal} title={translate('warning')}
+          yesAction={this.deleteServer}
+          noAction={() => this.setState({showDeleteServerConfirmModal: false})}>
+          {translate('server.delete.server.confirm', serverId)}
+        </YesNoModal>
+        <YesNoModal show={this.state.showImportServerConfirmModal} title={translate('warning')}
+          yesAction={this.saveImportedServers}
+          noAction={() => this.setState({showImportServerConfirmModal: false, importedResults: {}})}>
+          {this.props.mode && this.props.mode === 'addserver' ?
```

Comment:
again, unnecessary check before the &&

Reply:
same response as above...mode is not necessary 'addserver'

Reply:
ok will update